src/streamlink_cli/main.py

```diff
@@ -359,7 +359,7 @@ def handle_stream(plugin, streams, stream_name):
         try:
             console.msg("{0}", stream.to_url())
         except TypeError:
-            console.exit("The stream specified cannot be translated to a URL")
+            console.exit("The specified stream cannot be translated to a URL")
```

Comment:
`The specified stream cannot be translated into a URL.`
(translated into + full stop)